Makefile

```diff
@@ -35,7 +35,7 @@ flush_db : import_directory auth_models.json
 	# Reload country codes
 	python manage.py update_countries_plus
 
-recreate_db : import_directory flush_db import_docket_import
+recreate_db : import_directory flush_db import_docket_import download_to_s3
```

Comment:
I think it's pretty confusing to call this `download_to_s3`. How about `data_archive`?